plugin/mlsdbonlinelocator.cpp

```diff
@@ -94,21 +104,20 @@ bool MlsdbOnlineLocator::findLocation(const QList<MlsdbProvider::CellPositioning
         return true;
     }
 
-    if (!haveFieldData(cells)) {
-        // no field data available
-        qCDebug(lcGeoclueMlsdbOnline) << "No field data available for MLS online request";
-        return false;
-    }
-
-    QNetworkRequest req(QUrl("https://location.services.mozilla.com/v1/geolocate?key=" + m_mlsKey));
-    req.setHeader(QNetworkRequest::ContentTypeHeader, "application/json");
-
     QVariantMap map;
-    map.unite(globalFields());
     map.unite(cellTowerFields(cells));
     map.unite(wifiAccessPointFields());
+    if (map.isEmpty() && !m_fallbacksLacf && !m_fallbacksIpf) {
+        // no field data(cell, wifi) available
+       	qCDebug(lcGeoclueMlsdbOnline) << "No field data(cell, wifi) available for MLS online request";
```

Comment:
Whitespace issue, the qCDebug should indented to the same level as above comment. Do not use tabs.

Reply:
Ok